flame_fire_atlas/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [1.0.0-rc1]
```

Comment:
Missing version bump?

Reply:
Will do that after, I have some changes that I want to make after this PR before releasing

Reply:
So it should be [NEXT] now?